src/main/java/wolox/training/models/Book.java

```diff
@@ -62,6 +65,9 @@ public String getIsbn() {
     }
 
     public void setIsbn(String isbn) {
+        Preconditions.checkNotNull(isbn, String.format(ErrorConstants.NOT_NULL, "isbn"));
+        Preconditions.checkArgument(isbn.matches("^(0|[1-9][0-9]*)$\n"),
```

Comment:
I recommend using this library to do the check by numbers:

import org.apache.commons.lang.StringUtils; ---> import
StringUtils.isNumeric(isbn);